contracts/v2/sovereignChains/GlobalExitRootManagerL2SovereignChain.sol

```diff
@@ -16,6 +14,10 @@ contract GlobalExitRootManagerL2SovereignChain is
     // globalExitRootUpdater address
     address public globalExitRootUpdater;
 
+    // globalExitRootRemover address
+    // In case of initializing a chain with Full execution proofs, this address should be set to zero, otherwise, some malicious sequencer could insert invalid global exit roots, claim, go back and the execution would be correctly proved.
```

Comment:
What do you mean? I understood that in case of a FEP chain, this address has to be zero. Indeed, if it's not zero can be a multisig but what happens with the comment?